### DISCUSSION TITLE
[mpvnet] support for the English language

### DISCUSSION THREAD

**Discussion by SatL1Ng** [hooke007/mpv_PlayKit#258] [libmpv]
Could you please add support for the English language. I like your modification, but I don't understand Chinese. Thanks

能否请你增加英语语言支持。我喜欢你的修改，但我不懂中文。谢谢你

**Comment by hooke007** [comment]
There is no multi-language framework in mpvnet. Supporting other language is painful.
Considering that upstream would not maintain his project (https://github.com/mpvnet-player/mpv.net/issues/527), my fork will stop updating too.
